pages/mapping-introduction.md

```diff
@@ -45,11 +45,11 @@ Accurate mapping depends on high quality fieldwork. Ordinarily, fieldwork will u
 
 The fieldwork will usually be done at a larger scale than the final map. With convenient rescaling of computer graphic images it is now possible for fieldwork to be undertaken at any convenient scale such as 1:5000 or even larger. This demands less skill in field penmanship than the traditional fieldwork scale of 1:7500.
 
-The fieldwork is scanned &#8212; about 200dpi is usually quite sufficient resolution &#8212; now save the image as a .png file in your map file directory ( .jpg image files also work). This file is then loaded as a template underneath the new map screen and the features are inserted on the map on top of the fieldwork image. The fieldwork must be scaled and positioned appropriately for this purpose.
+The fieldwork is scanned - about 200dpi is usually quite sufficient resolution - now save the image as a .png file in your map file directory ( .jpg image files also work). This file is then loaded as a template underneath the new map screen and the features are inserted on the map on top of the fieldwork image. The fieldwork must be scaled and positioned appropriately for this purpose.
```

Comment:
Nitpick: &#8212;, i.e. "em dash", is typographically correct. Is there a technical problem with it (in the app or online)?

Reply:
Good question. I can't remember the reason but the first commits were done by Github's online editor.

Reply:
Okay, thanks. Qt Assistant limitation.